src/FlowLinesLayer/FlowLinesLayer.ts

```diff
@@ -128,11 +129,11 @@ class FlowLinesLayer extends Layer<Props, LayerState, Context> {
 
   draw({ uniforms }: DrawParams) {
     const { gl } = this.context;
-    const { borderColor } = this.props;
+    const borderColor = this.props.borderColor || DEFAULT_BORDER_COLOR;
     gl.lineWidth(1);
     this.state.model.render({
       ...uniforms,
-      borderColor: borderColor || DEFAULT_BORDER_COLOR,
+      borderColor: borderColor.map(x => (x * 1) / 255),
```

Comment:
what's the purpose of `x * 1`?

Reply:
prettier has actually broken this, it should be x => x * (1/255)

Reply:
which is the same as x / 255 of course.